tests/rptest/tests/datalake/mount_unmount_test.py

```diff
@@ -102,12 +102,15 @@ def test_simple_unmount(self, cloud_storage_type):
             config={"redpanda.iceberg.delete": "false"})
         connect = RedpandaConnectService(self.test_context, self.redpanda)
         connect.start()
-        verifier = DatalakeVerifier(self.redpanda, self.TOPIC_NAME,
-                                    self.dl.spark())
+        verifier = DatalakeVerifier(self.redpanda,
+                                    self.TOPIC_NAME,
+                                    self.dl.spark(),
+                                    buffer=50000)
```

Comment:
nit: maybe pull 50000 out into some `high_buffered_msgs` and explain in a comment why it's necessary?